solarforecastarbiter/reports/figures.py

```diff
@@ -680,13 +692,20 @@ def raw_report_plots(report, metrics):
                     figure_dict[f'{category}::{metric}::{name}'] = fig
     script, divs = components(figure_dict)
     mplots = []
+    try:
+        driver = _make_chrome_webdriver()
+    except Exception:  # pragma: no cover
```

Comment:
should I care about this? maybe a comment for why not.

Reply:
nope, just falls back to bokeh behaviour. Seemed hard to test properly without installing other os pkgs